src/bundles/files.js

```diff
@@ -241,17 +241,22 @@ export default (opts = {}) => {
         progress: updateProgress
       })
 
-      if (res.length !== streams.length + 1) {
+      if (res.length !== streams.length + 2) {
```

Comment:
Not sure when this changed, but it was always failing because of this!

Reply:
interesting, it's returning an entry for the folder, and the parent of the folder. This fixes our logic around it, so +1